cmd/blockchaincmd/create.go

```diff
@@ -358,6 +379,35 @@ func createBlockchainConfig(cmd *cobra.Command, args []string) error {
 	if err = promptValidatorManagementType(app, sc); err != nil {
```

Comment:
we just need logic to always enable native minter precompile is SoV, and add the manager as enabled, when generating evm params